server/controllers/overviewController.ts

```diff
@@ -114,7 +114,7 @@ export default class OverviewController {
         res.locals.apiErrorCallback,
       ),
       this.offencesService.getOffencesOverview(clientToken, bookingId, prisonerNumber),
-      this.offenderService.getPrisonerNonAssociationOverview(clientToken, prisonerNumber),
+      this.offenderService.getPrisonerNonAssociationOverview(clientToken, prisonerNumber, res.locals.apiErrorCallback),
```

Comment:
I notice that you've not gone down the Result path with this one, why was that? Whichever way we do it I'd rather it be consistent.